pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/rbac/leader_election_role.go

```diff
@@ -44,6 +45,11 @@ const leaderElectionRoleTemplate = `# permissions to do leader election.
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
+  labels:
+    app.kubernetes.io/name: leader-election-role
+    app.kubernetes.io/component: leader-election
```

Comment:
```suggestion
    app.kubernetes.io/component: rbac
```

Reply:
ihmo: The architecture component is RBAC or permissions  such as in the examples we have a database

Reply:
updated in `kustomize/v1` plugin in commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/8a2c1d7c48ec7743828b14a04ae9de765cde30b2

Reply:
Should be finished in both `kustomize` plugins as of commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/daaca98d6775fe54fda2a0017d2f0f4b1369c1de

Marking this particular comment as resolved.